lib/charms/mongodb/v0/helpers.py

```diff
@@ -19,7 +19,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 4
+LIBPATCH = 5
```

Comment:
**question:** What bumped the versions with no diff?

Reply:
CI: was complaining - must have forgotten to bump in a previous PR 😅